cpp/Makefile

```diff
@@ -189,10 +225,10 @@ $(XAUDIO27_TARGET) : directories $(XAUDIO27_OBJ) $(XAPO_TARGET)
 	$(CXX) $(CXXFLAGS) -shared -o $@ $(XAUDIO27_OBJ) $(XAPO_TARGET) $(LDFLAGS)
 
 $(XAUDIO28_TARGET) : directories $(XAUDIO28_OBJ) $(XAPO_TARGET)
-	$(CXX) $(CXXFLAGS) -shared -o $@ $(XAUDIO28_OBJ) $(XAPO_TARGET) $(LDFLAGS) -lsetupapi
+	$(CXX) $(CXXFLAGS) -shared -o $@ $(XAUDIO28_OBJ) $(XAPO_TARGET) $(XAPOFX15_TARGET) $(LDFLAGS) -lsetupapi
```

Comment:
For these two targets, add XAPOFX.cpp to XAUDIO2_SRC_x28. CreateFX is actually inside the XAudio2_8 dll itself, so this isn't quite the same.